src/mavlink_daemon.py

```diff
@@ -52,40 +52,43 @@
 
 
 while True:
-
-    # From vehicle to ground/tukano
-    vehicle_m = vehicle_link.recv()
-    vehicle_msgs = vehicle_link.mav.parse_buffer(vehicle_m)
-    if vehicle_msgs:
-        for vehicle_msg in vehicle_msgs:
-            logging.debug("(VEHICLE_MSG) {}".format(vehicle_msg))
-            if ground_link:
-                ground_link.write(vehicle_msg.get_msgbuf())
-
-            if tukano_link:
-                tukano_link.write(vehicle_msg.get_msgbuf())
-
-    # From ground to vehicle/tukano
-    if ground_link:
-        ground_m = ground_link.recv()
-        ground_msgs = ground_link.mav.parse_buffer(ground_m)
-        if ground_msgs:
-            for ground_msg in ground_msgs:
-                logging.debug("(GROUND_MSG) {}".format(ground_msg))
-                vehicle_link.write(ground_msg.get_msgbuf())
+    try:
+        # From vehicle to ground/tukano
+        vehicle_m = vehicle_link.recv()
+        vehicle_msgs = vehicle_link.mav.parse_buffer(vehicle_m)
+        if vehicle_msgs:
+            for vehicle_msg in vehicle_msgs:
+                logging.debug("(VEHICLE_MSG) {}".format(vehicle_msg))
+                if ground_link:
+                    ground_link.write(vehicle_msg.get_msgbuf())
 
                 if tukano_link:
-                    tukano_link.write(ground_msg.get_msgbuf())
-
-    # From tukano to vehicle/ground
-    tukano_m = tukano_link.recv()
-    tukano_msgs = tukano_link.mav.parse_buffer(tukano_m)
-    if tukano_msgs:
-        for tukano_msg in tukano_msgs:
-            logging.debug("(TUKANO_MSG) {}".format(tukano_msg))
-            vehicle_link.write(tukano_msg.get_msgbuf())
-
-            if ground_link:
-                ground_link.write(tukano_msg.get_msgbuf())
-
-    time.sleep(settings.SLEEPING_TIME)
+                    tukano_link.write(vehicle_msg.get_msgbuf())
+
+        # From ground to vehicle/tukano
+        if ground_link:
+            ground_m = ground_link.recv()
+            ground_msgs = ground_link.mav.parse_buffer(ground_m)
+            if ground_msgs:
+                for ground_msg in ground_msgs:
+                    logging.debug("(GROUND_MSG) {}".format(ground_msg))
+                    vehicle_link.write(ground_msg.get_msgbuf())
+
+                    if tukano_link:
+                        tukano_link.write(ground_msg.get_msgbuf())
+
+        # From tukano to vehicle/ground
+        tukano_m = tukano_link.recv()
+        tukano_msgs = tukano_link.mav.parse_buffer(tukano_m)
+        if tukano_msgs:
+            for tukano_msg in tukano_msgs:
+                logging.debug("(TUKANO_MSG) {}".format(tukano_msg))
+                vehicle_link.write(tukano_msg.get_msgbuf())
+
+                if ground_link:
+                    ground_link.write(tukano_msg.get_msgbuf())
+
+        time.sleep(settings.SLEEPING_TIME)
+
+    except:
+        pass
```

Comment:
Try to never leave empty except clauses:
* it's unclear what exception is being caught
* if an exception other than the one we are expecting is thrown, we don't want to catch it, we need the error to be shown

you can import `socket` module and catch the windows socket error (like `except socket.WSAEINVAL:` if I remember well)